configs/smartling-config.json

```diff
@@ -0,0 +1,102 @@
+{
+    "locales": [{
+            "smartling": "ar",
+            "application": "ar"
+        },
+        {
+            "smartling": "zh-CN",
+            "application": "zh-CN"
+        },
+        {
+            "smartling": "zh-TW",
+            "application": "zh-TW"
+        },
+        {
+            "smartling": "nl-NL",
+            "application": "nl"
+        },
+        {
+            "smartling": "fr-CA",
+            "application": "fr-CA"
+        },
+        {
+            "smartling": "fr-FR",
+            "application": "fr"
+        },
+        {
+            "smartling": "de-DE",
+            "application": "de"
+        },
+        {
+            "smartling": "hi-IN",
+            "application": "hi-IN"
+        },
+        {
+            "smartling": "id-ID",
+            "application": "id"
+        },
+        {
+            "smartling": "it-IT",
+            "application": "it"
+        },
+        {
+            "smartling": "ja-JP",
+            "application": "ja"
+        },
+        {
+            "smartling": "ko-KR",
+            "application": "ko"
+        },
+        {
+            "smartling": "ms-MS",
+            "application": "ms"
```

Comment:
For this initial landing, we only need the Malay setting. If those other languages remain, Smartling automation will create jobs for each of the languages listed here. All of the settings for the other languages can be removed. When we can confirm that Malay is working properly, we can update this with Arabic, Japanese, Hindi, and Indonesian.